src/raml1/artifacts/raml10parser.ts

```diff
@@ -1885,6 +1885,14 @@ setAdditionalProperties( param:boolean ){
 
 
         /**
+         * JSON schema style syntax for declaring key restricted maps
+         **/
+patternProperties(  ):TypeDeclaration[]{
```

Comment:
@ddenisenko @nnance we need to be careful here. There are no patternProperties in RAML 1.0. See https://github.com/raml-org/raml-js-parser-2/issues/363.